whipper/common/encode.py

```diff
@@ -89,3 +90,61 @@ def _tag(self):
         w.save()
 
         self.stop()
+
+
+class EmbedPictureTask(task.Task):
+    description = 'Embed picture to FLAC'
+
+    def __init__(self, track_path, coverart_path):
+        self.track_path = track_path
+        self.coverart_path = coverart_path
+
+    def start(self, runner):
+        task.Task.start(self, runner)
+        self.schedule(0.0, self._embed_picture)
+
+    def _make_flac_picture(self, coverart_filename):
+        """Given a path to a jpeg or png file, return a flac.Picture
+
+        Returns: a valid flac.Picture or None
+        """
+        if not coverart_filename:
+            return None
+
+        from PIL import Image
+
+        im = Image.open(coverart_filename)
+        if im.format == 'JPEG':
+            mime = 'image/jpeg'
+        elif im.format == 'PNG':
+            mime = 'image/png'
+        else:
+            logger.warning("no cover art will be added because the fetched "
```

Comment:
Nitpick, just to unify this warning message with the (proposed) other one:

```python
            logger.warning("no cover art will be added because the fetched "
                           "image format is unsupported")
```

Reply:
\o/ no problem.